lib/editing/editor.dart

```diff
@@ -186,10 +186,22 @@ class Completion {
   /// This can be `null`.
   final int cursorOffset;
 
+  /// Quick fixes that alter multiple lines of editor content can't simply
```

Comment:
nit: consider explaining what it is before the justification

Reply:
Good point. Fixed.